wandb/cli/cli.py

```diff
@@ -885,12 +887,12 @@ def _check_launch_imports():
     help="Version of the project to run, as a Git commit reference for Git projects.",
 )
 @click.option(
-    "--param-list",
-    "-P",
+    "--args-list",
+    "-a",
     metavar="NAME=VALUE",
     multiple=True,
-    help="A parameter for the run, of the form -P name=value. Provided parameters that "
-    "are not in the list of parameters for an entry point will be passed to the "
+    help="An argument for the entrypoint script, of the form -a name=value. Provided arguments that "
```

Comment:
nit but can we change this back to "run" instead of "entrypoint script" i think it's actually less confusing

Reply:
Sounds good